src/aleph_client/commands/instance/__init__.py

```diff
@@ -284,19 +295,17 @@ def validate_ssh_pubkey_file(file: Union[str, Path]) -> Path:
         item_hash: ItemHash = message.item_hash
 
         console = Console()
-        if crn and (confidential or not hold):
+        if crn and not hold:
             if not crn.url:
                 return
-            async with AuthenticatedAlephHttpClient(account=account, api_server=sdk_settings.API_HOST) as client:
```

Comment:
Why did you removed this first switch statement? It doesn't useful but I'm curious.

Reply:
I did remove it cause it was here but no used